pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -3402,6 +3488,454 @@ def SeismicActivityExceptionRule(model, k, t):
     return model
 
 
+def operational_hydraulics(model, opt):
+
+    if model.config.hydraulics == Hydraulics.post_process:
+        """
+        For the post process method, the pressure is computed in using
+        Hazen-Williams equation in a separate stand alone model. This model
+        consists of a pressure balance equation at each node and bounds for
+        pressure. The objective is to minimize the total pressure change (deltaP)
+        due to pumps and valves.
+        """
+        # Create a model and add hydraulics constraint to compute pressure at each node
+        mh = ConcreteModel()
+
+        mh.p_HW_material_factor_pipeline = Param(
+            initialize=130,
+            mutable=True,
+            units=pyunits.dimensionless,
+            doc="Pipeline material factor for Hazen Williams equation",
+        )
+        mh.v_PumpHead = Var(
+            model.s_LLA,
+            model.s_T,
+            within=NonNegativeReals,
+            initialize=0,
+            units=pyunits.meter,
+            doc="Pump Head added in the direction of flow, m",
+        )
+        mh.v_ValveHead = Var(
+            model.s_LLA,
+            model.s_T,
+            within=NonNegativeReals,
+            initialize=0,
+            units=pyunits.meter,
+            doc="Valve Head removed in the direction of flow, m",
+        )
+        mh.p_rhog = Param(
+            initialize=9.8 * 1000,
+            units=model.model_units["pressure"] / pyunits.meter,
+            doc="g (m/s2) * density of PW (assumed to be 1000 kg/m3)",
+        )
+        mh.p_Min_AOP = Param(
+            initialize=10 * 6895,
```

Comment:
I see `10` here and `14.7` a few liens below - should they match?

Reply:
The numbers have now been moved to the spreadsheet and should be the same everywhere

Reply:
Resolving.